### ISSUE TITLE
Excel text string NA is converted to an empty field by in2csv

### ISSUE THREAD

**Issue by martinburch** [wireservice/csvkit#1045]
## Input:
[testna.xlsx](https://github.com/wireservice/csvkit/files/3840255/testna.xlsx)

```
$ in2csv testna.xlsx
```

## Output, note Namibia's code is missing:
```
Country,Code
Namibia,
Nigeria,NG
```

## Expected output:
```
Country,Code
Namibia,NA
Nigeria,NG
```

**Comment by martinburch** [comment]
After reading the closed issues (#806 especially) I see this can be avoided by using `in2csv --blanks testna.xlsx` but it shouldn't be happening. The xlsx knows this is a text field. Other tools like `csvclean` could coerce the value by default, but in2csv mangling my text does not give me trust in this tool.

**Comment by jpmckinney** [comment]
The original authors and users of this tool worked with a lot of input files (whether Excel, CSV, etc.) where "NA" was used to mean "not applicable" and where it appeared in columns that were otherwise numbers, dates, etc. That's why the default behavior is to treat it as a blank value.

Since changing defaults is likely going to cause headaches for far more new and existing users than it's going to increase trust for new users, defaults are generally left as-is.
